src/server/gc_worker.rs

```diff
@@ -203,6 +207,80 @@ impl<E: Engine> GcRunner<E> {
         .map_err(Error::from)
     }
 
+    /// Check need gc without getting snapshot.
+    /// If this is not supported or any error happens, returns true to do further check after
+    /// getting snapshot.
+    fn need_gc(&self, ctx: &Context, safe_point: TimeStamp) -> bool {
+        let region_info_accessor = match &self.region_info_accessor {
+            Some(r) => r,
+            None => {
+                info!(
```

Comment:
Why log it?

Reply:
Only in tests the region_info_accessor and local_storage are none. I don't think it's needed to log it.

Reply:
In production it won't be printed. I just want to make it possible to know if `region_info_accessor` is incorrectly unspecified by viewing log.